test/media/drm_engine_integration.js

```diff
@@ -46,6 +46,8 @@ describe('DrmEngine', function() {
   var onKeyStatusSpy;
   /** @type {!jasmine.Spy} */
   var onExpirationSpy;
+  /** @type {!jasmine.Spy} */
+  var onEventSpy;
```

Comment:
I created this spy but don't actually use it for anything. Suggestions?

Reply:
I'm not sure we need a separate integration test for this, but you could always expect that onEvent was called at the appropriate time in the "basic flow" test below.

Note that DRM integration tests are only run if you use `./build/test.py --drm`.

Reply:
I've left this as is. The new behaviour is already covered by a unit test, and I noticed that `onExpirationSpy` is similarly unused in drm_engine_integration. I'll leave it to you guys to decide whether you need these spies.